api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -150,9 +281,8 @@ public String getStandardConceptIdColumn() {
     for (Domain d: domainList) {
       String query = "SELECT ";
       // VALUES HERE:
-      Optional<ValueSet> valueSetOpt = dataSet.getValues().stream()
-          .filter(valueSet -> valueSet.getDomain() == d)
-          .findFirst();
+      Optional<List<DomainValuePair>> valueSetOpt =  Optional.of(dataSet.getValues().stream()
```

Comment:
I don't think this needs to be an optional, and the check below can be `valueSetList.isEmpty()` (I think?)